### PR TITLE
ET-5168: Prebid Video support

### PR DESCRIPTION
**Pull request by anand-venkatraman**

### CODE REVIEW

File: modules/pulsepointBidAdapter.js [hunk 1]
```diff
@@ -109,6 +120,15 @@ function bidResponseAvailable(bidRequest, bidResponse) {
       if (idToImpMap[id]['native']) {
         bid['native'] = nativeResponse(idToImpMap[id], idToBidMap[id]);
         bid.mediaType = 'native';
+      } else if (idToImpMap[id].video) {
+        // for outstream, a renderer is specified
```
Comment (nikolay-neustroev) [inline modules/pulsepointBidAdapter.js]: consider extracting a method here

File: modules/pulsepointBidAdapter.md [hunk 2]
```diff
@@ -40,5 +40,57 @@ Please use ```pulsepoint``` as the bidder code.
               ct: 505642
           }
       }]
+    },{
+      code: 'outstream-div',
+      mediaTypes: {
+        video: {
+            playerSize: [640, 480],
```
Comment (nikolay-neustroev) [inline modules/pulsepointBidAdapter.md]: shouldn't outstream video size be adaptive i.e. width should be determined by content width and height should be determined by width and aspect ratio?

File: modules/pulsepointBidAdapter.md [hunk 3]
```diff
@@ -40,5 +40,57 @@ Please use ```pulsepoint``` as the bidder code.
...
+            context: 'outstream'
+        }
+      },
+      bids: [{
+          bidder: 'pulsepoint',
+          params: { 
+              cp: 512379,
+              ct: 505642
+              video: {
+                  h: 300,
```
Comment (nikolay-neustroev) [inline modules/pulsepointBidAdapter.md]: same as above - ideally size shouldn't be hardcoded

File: modules/pulsepointBidAdapter.md (hunk 2 above)
Comment (anand-venkatraman) [reply modules/pulsepointBidAdapter.md]: this is prebid spec, not ours

File: modules/pulsepointBidAdapter.md (hunk 3 above)
Comment (anand-venkatraman) [reply modules/pulsepointBidAdapter.md]: this is simply ortb.video object spec

**Pull request closed without merging**